integreat_cms/cms/views/pages/page_bulk_actions.py

```diff
@@ -183,3 +186,40 @@ def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
 
         # Let the base view handle the redirect
         return super().post(request, *args, **kwargs)
+
+
+class CancelTranslationProcess(PageBulkActionMixin, BulkActionView):
+    """
+    Bulk action to cancel translation process
+    """
+
+    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        r"""
+        Function to cancel the translation process for multiple pages of the current language at once
+
+        :param request: The current request
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        :return: The redirect
+        """
+
+        language_slug = kwargs["language_slug"]
+        succes = []
```

Comment:
```suggestion
        success = []
```
Nit: A small typo - and I'd probably call it `success_messages`, but I can live with `success` too :)